11ty/es/definiciones/no-binarie.md

```diff
@@ -0,0 +1,17 @@
+---
+title: No binarie
+slug: no-binarie
+speech: adj
+defined: true
+excerpt: de, en relación con, o caracterizade por cualquier género que no encaje en el binarismo de género másculino-femenino; un término paraguas.
```

Comment:
Here it should read "masculino" and not "másculino". Also, this is my first proofread for this project, and I am assuming it was agreed by the core team to use -e endings for gendered words? e.g. "caracterizade" instead of "caracterizado/a", or "binarie" instead of "binario/a"? Just making sure this is generally agreed upon. Thanks!

Reply:
Thank you. I fixed the typo. Regarding the use of -e ending, it was suggested by @tatianamac in the Discord server.